tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -372,6 +372,55 @@ create_disk() {
 	OK "Partitions created"
 }
 
+create_erofs_rootfs_image() {
```

Comment:
Have you tested DAX with erofs?

Reply:
For kata containers, nope yet. Since EROFS fsdax feature was just upstreamed for this merge window, it was tested independently (but I didn't tested it with kata containers.)

Reply:
Also note that their is still some WIP feature for fsdax --- is that EROFS fsdax tail-packing inline files are still under development due to the tail block is inlined in the metadata block and increase the complexity.
Tail-packing inline files can be disabled by "-Enoinline_data" for now. I will address "EROFS fsdax tail-packing inline files" in the future.

Anyway, it's just a head up and new WIP topic for now. I'll complete it later in my extra time.

(Feel free to add "do-not-merge" as well.)

Reply:
thanks @hsiangkao